test/CMakeLists.txt

```diff
@@ -50,6 +60,16 @@ else()
   set(GTEST_BOTH_LIBRARIES "gtest;gtest_main")
 endif()
 
+if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
+  set_property(
+      DIRECTORY
```

Comment:
The indent is not right.

Reply:
Fixed in https://github.com/WasmEdge/WasmEdge/pull/2751/commits/d61383c921e44f4f0f7bf71de3c3a20a67f49684